ev3dev/core.py

```diff
@@ -955,6 +962,91 @@ def wait_while(self, s, timeout=None):
         """
         return self.wait(lambda state: s not in state, timeout)
 
+    def _set_position_rotations(self, max_speed_pct, rotations):
```

Comment:
Why `max_speed_pct`? I read that to mean "the maximum speed (percentage)" as opposed to "percentage of the max speed". TBH, I think it's an unnecessary distinction as most people would just consider it "speed percentage".

Reply:
How about 'speed_pct'?

Reply:
I like that.